tests/wpt/web-platform-tests/webxr/xrWebGLLayer_constructor.https.html

```diff
@@ -17,8 +17,11 @@
   let gl = webglCanvas.getContext('webgl', glAttributes);
   return navigator.xr.test.simulateDeviceConnection(TRACKED_IMMERSIVE_DEVICE)
     .then(() => {
-      return navigator.xr.requestSession('inline')
-        .then((session) => {
+      let sessionPromise;
+      navigator.xr.test.simulateUserActivation(function() {
+        sessionPromise = navigator.xr.requestSession('inline');
```

Comment:
Note: this was a mistake, inline sessions with no tracking data (which must be explicitly requested via features) don't need activation

Reply:
File an issue? That sounds like a mistake in our implementation.

Reply:
In other words, we don't have enough of an implementation of the prerequisites to write the code for this. Paul's implementation is correct for the features we have right now, it's just the test change that isn't. Alex Cooper is already fixing the test so we're fine, I just wanted to note this.